site/styles/_mixins.scss

```diff
@@ -23,3 +23,15 @@
     z-index: 8;
     @include gradient-x( rgba($left-gradient-bg-color, 1), transparent, 0%, 100%);
 }
+
+@mixin a-links{
+    color: $body-color;
+    text-decoration: underline;
+    background-color: transparent;
+    font-weight: bold;
```

Comment:
The underline is enough, don't need to be bold.

```suggestion
```

Reply:
FYI - We use these variables for font weights now:
`$font-weight-light: 300 !default;
$font-weight-normal: 400 !default;
$font-weight-bold: 500 !default;`